### PR TITLE
Update EDT to version 2.8.2

### PR DESCRIPTION
**Pull request by fullofcaffeine**
### Changes proposed in this Pull Request

* Version bump and changelog

### Testing instructions

* Make sure that the line-height UI option appears (see: https://github.com/Automattic/wp-calypso/pull/46873)

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/readme.txt [hunk 1]
```diff
@@ -40,7 +40,10 @@ This plugin is experimental, so we don't provide any support for it outside of w
 
 == Changelog ==
 
-= 2.8.1 = 
+= 2.8.2 =
+* Gutenberg: Make line-height setting more robust. (https://github.com/Automattic/wp-calypso/pull/46873)
```
Comment (ockham) [inline apps/editing-toolkit/editing-toolkit-plugin/readme.txt:44]: Let's include that we actually fixed the line-height setting (through a different PR):

```suggestion
* Gutenberg: Support new line height setting (https://github.com/Automattic/wp-calypso/pull/46792)
* Gutenberg: Make line-height setting more robust. (https://github.com/Automattic/wp-calypso/pull/46873)
```

**Review by ockham — CHANGES_REQUESTED** [review]
Left one note about a [Changelog entry](https://github.com/Automattic/wp-calypso/pull/46887/files#r513807309). Otherwise good to go!

**Review by ockham — APPROVED** [review]
Nice :+1:

**Review by wp-desktop — CHANGES_REQUESTED** [review]
WordPress Desktop CI Failure for job "wp-desktop-mac".

@fullofcaffeine please inspect this job's build steps for breaking changes at [this link](https://circleci.com/gh/Automattic/wp-calypso/936623). For temporal failures, you may try to "Rerun Workflow from Failed".

Please also ensure this branch is rebased off latest Calypso.

**Pull request merged**